Week2/README.md

```diff
@@ -15,10 +15,4 @@ Here are resources that we like you to read as a preparation for the second lect
 - Read about Asynchronous vs. Synchronous programming: http://www.hongkiat.com/blog/synchronous-asynchronous-javascript/
 - [Why closures are helpful with async code](http://stackoverflow.com/questions/13343340/calling-an-asynchronous-function-within-a-for-loop-in-javascript)
 
-### Fundamentals
```

Comment:
@grod220 was these removed by mistake? Or is there something I miss?

Reply:
I'm pretty sure the links didn't work 🤔

Reply:
They work assuming you have forked the fundamentals repo (atleast that’s what I found)

Reply:
Added a PR to restore them, #5